fr/sidebars.json

```diff
@@ -173,11 +173,29 @@
             "monitoring/event-handler",
             "monitoring/web-import-export"
         ],
-        "Alertes et Notifications": [
+        "Gérer évènements et alertes": [    
+            {
+                "ids":[
             "alerts-notifications/concepts",
             "alerts-notifications/resources-status",
             "alerts-notifications/event-console",
-            "alerts-notifications/manage-alerts",
+            "alerts-notifications/event-log",
+            "alerts-notifications/custom-views"             
+                ],
+                "label": "Visualiser les évènements",
+                "type": "subcategory"
+            },  
+            {   
+                "ids" : [
+                    "alerts-notifications/acknowledge",
+                    "alerts-notifications/downtimes",
+                    "alerts-notifications/submit",
+                    "alerts-notifications/notif-flapping",
```

Comment:
This is both times, first here but no present into table of contant and secondly in "Gérer les notifications"